.github/workflows/format.yml

```diff
@@ -0,0 +1,42 @@
+name: format-check
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release-'
+      - 'develop'
```

Comment:
Why dont we run this on all branches? Just noticed this, sorry.. lets have this Action run on all branches for every push